cla_backend/apps/call_centre/serializers.py

```diff
@@ -59,7 +59,7 @@ class PartnerIncomeSerializer(IncomeSerializerBase):
     Like IncomeSerializer but without 'child_benefits'
     """
 
-    self_employed = serializers.NullBooleanField(default=None)
+    self_employed = serializers.NullBooleanField(default=False)
```

Comment:
@BenMillar-CGI is there any reason we changed this from defaulting from None to False?